sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_helpers.cs

```diff
@@ -22,11 +23,11 @@ public partial class WebPubSubServiceClient
         /// Creates a URI with authentication token.
         /// </summary>
         /// <returns></returns>
-        public virtual Uri GetClientAccessUri(string userId = default, string[] roles = default, TimeSpan expireAfter = default)
+        public virtual Uri GenerateClientAccessUri(string userId = default, string[] roles = default, TimeSpan expiresAfter = default)
```

Comment:
Nit - add a doc comment that `expiresAfter` defaults to one hour if not provided.